site/_plugins/jekyll-mermaid-block.rb

```diff
@@ -0,0 +1,16 @@
+module Jekyll
+  class Mermaid < Liquid::Block
+
+    def initialize(tag_name, markup, tokens)
+      super
+    end
+
+    def render(context)
+      @config = context.registers[:site].config['mermaid']
+      "<script src=\"/assets/js/mermaid.min.js\"></script>"\
```

Comment:
Let's add it here: https://github.com/FusionAuth/fusionauth-site/blob/master/site/_includes/_head.liquid#L97 so we load it only once rather than loading it every time.